opm/output/Cells.hpp

```diff
@@ -32,6 +32,10 @@ class SimulationDataContainer;
 namespace data {
 
     struct CellData {
+        CellData(std::string name, UnitSystem::measure dim, std::vector<double> data)
```

Comment:
> I got a compilation error

Interesting, I wonder why?

In any case adding the constructor is a good enough workaround. I agree we might want to use an rvalue-reference for `data` though.